src/os/filestore/FileStore.cc

```diff
@@ -2939,12 +2939,6 @@ void FileStore::_do_transaction(
       assert(0 == "collection attr methods no longer implemented");
       break;
 
-    case Transaction::OP_STARTSYNC:
-      tracepoint(objectstore, startsync_enter, osr_name);
-      _start_sync();
-      tracepoint(objectstore, startsync_exit);
-      break;
-
```

Comment:
Now FileStore::_start_sync() is not used anywhere. So that also need to be dropped ?